attestation-service/Cargo.toml

```diff
@@ -41,9 +41,11 @@ async-trait.workspace = true
 base64.workspace = true
 cfg-if.workspace = true
 clap = { workspace = true, optional = true }
+ear.workspace = true
 env_logger = { workspace = true, optional = true }
 futures = "0.3.17"
 hex.workspace = true
+jsonwebtoken = "9.3.0"
```

Comment:
`jsonwebtoken` is imported in workspace Cargo.toml. We can directly `jsonwebtoken.workspace = true` here